tests/test_script.py

```diff
@@ -6,7 +7,8 @@
 
 DIR = os.path.dirname(os.path.abspath(__file__))
 REPO = os.path.dirname(DIR)
-WHITELIST = os.path.join(REPO, 'vulture', 'whitelists', 'stdlib.py')
+WHITELIST_GLOB = os.path.join(REPO, 'vulture', 'whitelists', '*.py')
+WHITELISTS = glob.glob(WHITELIST_GLOB)
```

Comment:
I'd prefer globbing. You can combine the two lines above. No need to keep a reference to the glob expression around.